examples/simple_arm_motion/README.md

```diff
@@ -52,15 +52,16 @@ def hello_arm(config):
 ```
 If you want to ensure you only communicate with the robot via the ROS2 driver (which is not necessary, but may simplify your life and ensures all robot commands can be echoed on ROS topics, are caught in ros bags, etc), the best way to do so is to ensure that you never call `authenticate` in any other program.  That is done by the Spot driver, which should be the only piece of code that communicates with the robot.  All other programs communicate with the Spot driver via ROS2.  Therefore, we replace the pieces of code that talk directly to Spot with their ROS counterparts:
 ```python
-    node = Node('arm_simple')
-    tf_listener = TFListenerWrapper('arm_simple_tf', wait_for_transform = [ODOM_FRAME_NAME,
-                                                                           GRAV_ALIGNED_BODY_FRAME_NAME])
+    node = ros_scope.node()
+
+    tf_listener = TFListenerWrapper(node)
+    tf_listener.wait_for_a_tform_b(ODOM_FRAME_NAME, GRAV_ALIGNED_BODY_FRAME_NAME)
 
     robot = SimpleSpotCommander()
-    robot_command_client = ActionClientWrapper(RobotCommand, 'robot_command')
+    robot_command_client = ActionClientWrapper(RobotCommand, 'robot_command', node)
 ```
 This gives us four components, which we'll use in many ROS2 programs:
-* A node: [ROS2 nodes](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Nodes/Understanding-ROS2-Nodes.html) are the objects that interact with [ROS2 topics](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Topics/Understanding-ROS2-Topics.html) and almost all ROS programs require them.
+* A node: [ROS2 nodes](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Nodes/Understanding-ROS2-Nodes.html) are the objects that interact with [ROS2 topics](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Topics/Understanding-ROS2-Topics.html) and almost all ROS programs require them. Here we fetch the one in scope, already instantiated and serviced in the background. Had we instantiated a node of our own, we would have had to spinning ourselves.
```

Comment:
nit/typo: "spin it" ourselves?

Reply:
Fixed in https://github.com/bdaiinstitute/spot_ros2/pull/150/commits/56605892d07d8f438b550bb7e296cff966fd029f.